### PR TITLE
[NEW] Continuous sound notifications for new LiveChat rooms

### LINKED ISSUE #9847: Continuous notification (beep) for new live chats
### Description

- [x] add continuous notification (beep) for new live chats
- [x] create a setting to enable/disable the feature (disabled by default)

### PR DESCRIPTION
**Pull request by renatobecker**
@RocketChat/core 

Closes #9847 

This PR adds a new feature that allows to send continuous sound notifications when a new Livechat message arrives.
This means that the Livechat agent will hear sound notifications while there is a room with at least one unread message.

### CODE REVIEW

**Comment by renatobecker** [comment]
Fixed PR review.

File: packages/rocketchat-livechat/client/startup/notifyUnreadRooms.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+				fields: {
+					'settings.preferences.newRoomNotification': 1
+				}
+			});
+
+			const newRoomNotification = RocketChat.getUserPreference(user, 'newRoomNotification');
+			const [audio] = $(`audio#${ newRoomNotification }`);
+			if (audio && audio.play) {
+				if (unreadAlertCount > 0) {
+					Meteor.setTimeout(() => {
```
Comment (geekgonecrazy) [inline packages/rocketchat-livechat/client/startup/notifyUnreadRooms.js]: @rocketchat/core does anyone know... If this isn't unset does it keep executing?  Or does the meteor wrapper here work just like normal setTimeout?

**Comment by renatobecker** [comment]
There is an issue in this PR that needs to be reviewed.
When a new room is created, the sound notification is played correctly.
But if another room gets focus, even if there are unread rooms, the sound is paused.
So I'm going to work on this to make it work as expected.

**Comment by renatobecker** [comment]
New room notification was being paused when any room got focus.
This was happening because of this:

https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-ui/client/lib/notification.js#L123

Now it's fixed.

**Comment by ggazzo** [comment]
too* :)

**Comment by ggazzo** [comment]
:kissing_heart:

**Pull request merged**